src/components/FileAnnotation/SelectElement.js

```diff
@@ -1,42 +1,45 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import classes from 'classnames';
 import {fileTypeDetail} from '../../common/fileUtils';
-import {Icon} from 'semantic-ui-react';
+import {Icon, Radio, Segment, Header} from 'semantic-ui-react';
 /**
  * A radio button that displays information about a file type with a title,
  * description, and icon.
  */
-const SelectElement = ({className, value, select, selected}) => {
-  let selectElementClass = classes(
-    'SelectElement',
-    selected && 'SelectElement-fill',
-    className,
-  );
+const SelectElement = ({value, select, selected}) => {
   return (
-    <label className={selectElementClass}>
-      <input
-        className="border m-16"
-        type="radio"
-        name={value}
-        value={value}
-        checked={selected}
-        onChange={e => select(e)}
-      />
-      <Icon
-        name={`${fileTypeDetail[value].icon || 'question'}`}
-        size="large"
-        bordered
-        circular
-        inverted
-        color={selected ? 'blue' : null}
-      />
-      <div>
-        <p className="m-0 font-bold font-sm">{fileTypeDetail[value].title}</p>
-        <span className="font-normal text-grey text-xs">
-          {fileTypeDetail[value].description}
-        </span>
-      </div>
+    <label>
+      <Segment
+        compact
+        clearing
+        color={selected && 'blue'}
+        className="selection--card"
+      >
+        <Header as="h5" floated="left">
+          <Radio
+            name={value}
+            value={value}
+            checked={selected}
+            onChange={e => select(e)}
```

Comment:
add `label` as a render prop and pass the icon to it, this will replace the default browser checkbox with the icon 
```
label={()=> ( icon goes here))
```